medicines/search-client/src/models.rs

```diff
@@ -80,3 +91,48 @@ pub struct IndexEntry {
     pub substance_name: Vec<String>,
     pub facets: Vec<String>,
 }
+
+// The AzureResult model does not contain all of the information we want in the index,
```

Comment:
comment is out-of-date with the `AzureResult` -> `IndexResult` rename

Reply:
Updated.